main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       hand
+//  action 1:   the hand opens
+hand.open();
+//  action 2:   the hand waves
+hand.wave();
+//  action 3:   the hand makes a fist
+hand.makeFist();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       eyes
+//  action 1:   the eyes blink
+eyes.blink();
+//  action 2:   the eyes close
+eyes.close();
+//  action 3:   the eyes open wide
+eyes.openWide();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       horse
+//  action 1:   the horse trots
+horse.trot();
+//  action 2:   the horse gallops
+horse.gallop();
+//  action 3:   the horse bucks 
+horse.buck();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.